modules/clients/src/test/java/org/apache/ignite/internal/processors/rest/ChangeStateCommandHandlerTest.java

```diff
@@ -87,23 +89,23 @@
     public void testActivateDeActivate() throws GridClientException {
         GridClientClusterState state = client.state();
 
-        boolean active = state.active();
+        boolean active = state.state() == ACTIVE;
```

Comment:
`ClusterState.active(ClusterState state)` was used.

Reply:
> `ClusterState.active(ClusterState state)` was used.

`GridClientClusterState.active()` was used, no? Changed to `GridClientClusterState.state()`. Isn't is ok?